Description: Set loging level to INFO instead of TRACE.
Author: Viacheslav Chimishuk <vchimishuk@yandex.ru>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: kafka-2.2.1/config/log4j.properties
===================================================================
--- kafka-2.2.1.orig/config/log4j.properties
+++ kafka-2.2.1/config/log4j.properties
@@ -77,13 +77,13 @@ log4j.additivity.kafka.request.logger=fa
 log4j.logger.kafka.network.RequestChannel$=WARN, requestAppender
 log4j.additivity.kafka.network.RequestChannel$=false

-log4j.logger.kafka.controller=TRACE, controllerAppender
+log4j.logger.kafka.controller=INFO, controllerAppender
 log4j.additivity.kafka.controller=false

 log4j.logger.kafka.log.LogCleaner=INFO, cleanerAppender
 log4j.additivity.kafka.log.LogCleaner=false

-log4j.logger.state.change.logger=TRACE, stateChangeAppender
+log4j.logger.state.change.logger=INFO, stateChangeAppender
 log4j.additivity.state.change.logger=false

 # Access denials are logged at INFO level, change to DEBUG to also log allowed accesses
